src/templates/components/CapiHostedCard.svelte

```diff
@@ -0,0 +1,131 @@
+<script context="module" lang="ts">
+	import { clickMacro } from '$lib/gam';
+</script>
+
+<script lang="ts">
+	import AudioIcon from './icons/AudioIcon.svelte';
+	import CameraIcon from './icons/CameraIcon.svelte';
+	import VideoIcon from './icons/VideoIcon.svelte';
+
+	export let card;
+
+	const { headline, image, url, audioTag, galleryTag, videoTag } = card;
+
+	const pictureSupported =
+		image?.sources?.length > 0 && 'srcset' in new Image();
+</script>
+
+<a href={clickMacro(url)}>
```

Comment:
I think there's been a mix up here with the url - the link that's currently being added here is not the article link, it's the URL that we give to CAPI. This needs to be changed to the article link.

Currently:
<img width="145" alt="Screenshot 2024-01-11 at 10 53 28" src="https://github.com/guardian/commercial-templates/assets/108270776/3cec217b-cb12-4ded-ac00-8d7b1af30fc1">
Should be:
<img width="403" alt="Screenshot 2024-01-11 at 10 54 23" src="https://github.com/guardian/commercial-templates/assets/108270776/e71b5e5e-c95c-405a-82d9-69fe5ff92878">

Reply:
Nice catch. Have updated